libs/types/src/dtos/account/accounts.response.dto.ts

```diff
@@ -1,59 +1,49 @@
 /* eslint-disable max-classes-per-file */
-import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 import { IsNotEmpty, IsOptional } from 'class-validator';
 import { HexString } from '@polkadot/util/types';
 import { IsHexValue } from '#utils/decorators';
 import { IsAccountIdOrAddress } from '#utils/decorators/is-account-id-address.decorator';
 
 export class HandleResponseDto {
-  @ApiProperty()
```

Comment:
I'll test--but I don't think the `@ApiProperty` decorator counts as far as this is concerned; it's only for metadata, not any kind of payload validation.

However, the reason there are no other decorators is because this is a *response* object only--the ONLY reason to make it a concrete DTO class rather than a simple interface or typedef is so that the Swagger/OpenAPI generation will work on it to document the response payload properly.

Reply:
👇🏻 same with the other classes in this file